Makefile

```diff
@@ -216,30 +230,69 @@ vald/sha/print:
 
 .PHONY: vald/sha/update
 ## update VALD_SHA value
-vald/sha/update: vald
-	(cd vald; git rev-parse HEAD > ../$(VALD_SHA))
+vald/sha/update: $(VALD_DIR)
+	(cd $(VALD_DIR); git rev-parse HEAD | tr -d '\n' > ../$(VALD_SHA))
 
-.PHONY: vald/client/node/version/print
-## print VALD_CLIENT_NODE_VERSION value
-vald/client/node/version/print:
+.PHONY: vald/client/version/print
+## print VALD_CLIENT_JAVA_VERSION value
```

Comment:
```suggestion
## print VALD_CLIENT_NODE_VERSION value
```

Reply:
Thank you for catching it! 🙏  I apply your suggestion!